ensure __jump_table is parsed and we can tell that it effectively didn't change

diff -Nupr src.orig/kernel/time/timer.c src/kernel/time/timer.c
--- src.orig/kernel/time/timer.c	2017-11-17 15:58:51.501211972 -0500
+++ src/kernel/time/timer.c	2017-11-17 15:59:42.654211972 -0500
@@ -1643,6 +1643,9 @@ static __latent_entropy void run_timer_s
 	 */
 	base->must_forward_clk = false;
 
+	if (!base)
+		printk("kpatch-test: testing __tracepoints section changes\n");
+
 	__run_timers(base);
 	if (IS_ENABLED(CONFIG_NO_HZ_COMMON) && base->nohz_active)
 		__run_timers(this_cpu_ptr(&timer_bases[BASE_DEF]));
